src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -54,12 +54,19 @@ export default function StatusAlert({ appointment, facility }) {
         <strong>We’ve scheduled and confirmed your appointment.</strong>
         <br />
         <div className="vads-u-margin-y--1">
-          <Link to="/" onClick={handleClick()}>
-            Review your appointments
-          </Link>
+          <va-link
+            href="/"
```

Comment:
This link is going to the wrong path. I think "/" was enrolled in a special routes file. @vbahinwillit may have the background on it.